www/docs.md

```diff
@@ -100,9 +100,12 @@ The core feature of kutty is a set of attributes that allow you to issue AJAX re
 
 * [kt-get](/attributes/kt-get) - Issues a `GET` request to the given URL
 * [kt-post](/attributes/kt-post) - Issues a `POST` request to the given URL
-* [kt-put](/attributes/kt-put) - Issues a `PUT` request to the given URL (see [details](#kutty-request-details))
-* [kt-patch](/attributes/kt-patch) - Issues a `PATCH` request to the given URL  (see [details](#kutty-request-details))
-* [kt-delete](/attributes/kt-delete) - Issues a `GET` request to the given URL (see [details](#kutty-request-details))
+* [kt-put](/attributes/kt-put) - Issues a `PUT` request to the given URL
+* [kt-patch](/attributes/kt-patch) - Issues a `PATCH` request to the given URL
+* [kt-delete](/attributes/kt-delete) - Issues a `DELETE` request to the given URL
+
+(Since most browsers only support issuing `GET` and `POST`, a request with one of the other three methods will
+actually be issued as a `POST`, with the `X-HTTP-Method-Override` header set to the desired method.)
```

Comment:
There doesn't seem to be a kutty-request-details anchor, so I figured these links could be removed. Then I thought, what's special about these three methods? And I guessed it's the use of X-HTTP-Method-Override, so I borrowed this text from the method attribute pages and generalized it.